analyses/molecular-subtyping-pathology/03-incorporate-pathology-feedback.Rmd

```diff
@@ -316,6 +323,10 @@ compiled_df <- compiled_df %>%
     molecular_subtype = case_when(
       sample_id == "7316-1102" ~ "DMG, H3 K28",
       TRUE ~ molecular_subtype
+    ),
+    Notes = case_when(
+      sample_id == "7316-1102" ~ "Updated via OpenPBTA subtyping",
```

Comment:
I don't quite understand why it looks like there's recoding by individual samples -- it would probably be good to look into a more systematic way to do this, but that should be in a different PR, not now.

Reply:
This notebook is designed to handle individual cases based on feedback from pathologists / clinical expertise. I think it's possible that coding for individual cases with lots of context is more clear.